src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs

```diff
@@ -628,7 +642,7 @@ private unsafe int HandleEventShutdownComplete(ref SHUTDOWN_COMPLETE_DATA data)
             _receiveTcs.TrySetException(exception, final: true);
             _sendTcs.TrySetException(exception, final: true);
         }
-        _startedTcs.TrySetResult();
+        _startedTcs.TrySetException(ThrowHelper.GetOperationAbortedException());
```

Comment:
Should we also unset `_decrementStreamCapacity` here? 🤔
Or I guess it might be related to my question above re: `_decrementStreamCapacity` and `_startedTcs` alignment

Reply:
It'll be already unset here. We set it when we call MsQuic `StreamStart` ==> we'll get `START_COMPLETE` 100% where we unset it. But the place I overlooked is when we get an error from `StreamStart`, I'll fix it.